pkl-core/src/main/java/org/pkl/core/project/Package.java

```diff
@@ -25,113 +25,93 @@
 
 /** Java representation of class {@code pkl.Project#Package} */
 @SuppressWarnings("unused")
-public final class Package {
-
-  private final String name;
-  private final PackageUri uri;
-  private final Version version;
-  private final URI packageZipUrl;
-  private final @Nullable String description;
-  private final List<String> authors;
-  private final @Nullable URI website;
-  private final @Nullable URI documentation;
-  private final @Nullable URI sourceCode;
-  private final @Nullable String sourceCodeUrlScheme;
-  private final @Nullable String license;
-  private final @Nullable String licenseText;
-  private final @Nullable URI issueTracker;
-  private final List<Path> apiTests;
-  private final List<String> exclude;
-
-  public Package(
-      String name,
-      PackageUri uri,
-      Version version,
-      URI packageZipUrl,
-      @Nullable String description,
-      List<String> authors,
-      @Nullable URI website,
-      @Nullable URI documentation,
-      @Nullable URI sourceCode,
-      @Nullable String sourceCodeUrlScheme,
-      @Nullable String license,
-      @Nullable String licenseText,
-      @Nullable URI issueTracker,
-      List<Path> apiTests,
-      List<String> exclude) {
-    this.name = name;
-    this.uri = uri;
-    this.version = version;
-    this.packageZipUrl = packageZipUrl;
-    this.description = description;
-    this.authors = authors;
-    this.website = website;
-    this.documentation = documentation;
-    this.sourceCode = sourceCode;
-    this.sourceCodeUrlScheme = sourceCodeUrlScheme;
-    this.license = license;
-    this.licenseText = licenseText;
-    this.issueTracker = issueTracker;
-    this.apiTests = apiTests;
-    this.exclude = exclude;
-  }
-
+public record Package(
```

Comment:
We can remove `equals`, `hashCode`, and `toString` for this one right?

Reply:
I've removed `equals` and `hashCode`. The `toString` function on `record` uses square brackets `MyObjectName[...]` whereas this one uses curly braces `Package{...}`. What would you prefer to do here?

Reply:
It's fine to just use the one on `record`; the `toString()` is only meant for debugging purposes anyway!